sinon/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for Sinon 1.16.0
+// Type definitions for Sinon 2.1.0
 // Project: http://sinonjs.org/
-// Definitions by: William Sears <https://github.com/mrbigdog2u>
+// Definitions by: William Sears <https://github.com/mrbigdog2u>, Jonathan Little <https://github.com/rationull>
```

Comment:
Maybe adding attribution isn't actually appropriate for such a small change but I'm not sure what the standards are for this...